assets/css/fonts.scss

```diff
@@ -6,7 +6,9 @@
   font-family: "Material Icons";
   font-style: normal;
   font-weight: 400;
-  font-display: swap;
+  // Do not use font-display: swap for the icon font - it looks really bad when the page
+  // loads.
+  font-display: block;
```

Comment:
Oops! I realized this was the reason for #462 
Thank you @NBTX, I never knew this!